### PR TITLE
significantDigits fallback and lisk logo

### PR DESCRIPTION
**Pull request by Klizzy**
If i have 3 currencies specified in the config (e.g `currency['bitcoin', 'ethereum', ''lisk]`), but only the default value of `significantDigits [2]` every following price of the next currency has the price of `NaN`.Before this fix it would have only had worked if i had the significantDigits for every currency `significantDigits[2,2,2]`.

This fix adds a fallback for a otherwise breaking change if someone updates from an older version.
Additionally it makes it easier for the initial setup, because now it still works as intended if more than one currency is specified.

Basically this commit just sets the default significantDigit if its not accurately defined for every single one.

Lastly i added the lisk logo in both variants. 

edit:
i saw that there is even a issue already to it :D 
This pull request fixes this issue: #46

### CODE REVIEW

**Comment by matteodanelli** [comment]
Thanks klizzy!
